function-app/adb-to-purview/src/Function.Domain/Helpers/PurviewCustomType.cs

```diff
@@ -559,8 +571,13 @@ public async Task<EntityModel> search_entities(string QualifiedName, string type
         {
             var correlationId = Guid.NewGuid().ToString();
             var token = this.GetToken();
-            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.catalog.purview.azure.com/api";
-            var purviewSearchEndpoint = $"{baseurl}/atlas/v2/search/advanced";
+
+            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.{Environment.GetEnvironmentVariable("ResourceUri") ?? "purview.azure.com"}";
```

Comment:
This breaks anyone's existing appsettings since the current default is https://purview.azure.com. Should this be a constant variable instead of the string being repeated?